### PR TITLE
[Merged by Bors] - feat(NumberTheory/NumberField/Completion): API for `InfinitePlace ℚ`

### PR DESCRIPTION
**Pull request by kbuzzard**
From the FLT project.

When trying to do some calculations on adeles for FLT I realised that the completion of `ℚ` at its infinite place was not equal to `ℝ` and moreover we basically knew nothing about it. After this PR we at least know which rationals are in its open unit ball (which was what I needed). 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/NumberTheory/NumberField/Embeddings.lean [hunk 1]
```diff
@@ -1066,3 +1069,29 @@ theorem nrRealPlaces_eq_zero_of_two_lt (hk : 2 < k) (hζ : IsPrimitiveRoot ζ k)
...
+open NumberField
+
+noncomputable def infinitePlace : InfinitePlace ℚ := .mk (Rat.castHom _)
+
+instance : Subsingleton (InfinitePlace ℚ) where
+  allEq a b := Subtype.ext <| by
+    obtain ⟨_, _, rfl⟩ := a
+    obtain ⟨_, _, rfl⟩ := b
+    ext
+    simp
```
Comment (erdOne) [inline Mathlib/NumberTheory/NumberField/Embeddings.lean:1090]: If you add an appropriate ext lemma and tag `infinitePlace_apply` as simp then this is just `by ext; simp` or `by aesop`.

Comment (kbuzzard) [reply Mathlib/NumberTheory/NumberField/Embeddings.lean]: Should `infinitePlace_apply` be `simp`? I added the `ext` lemma.

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**